main.go

```diff
@@ -121,9 +121,10 @@ var (
 	metricsInterval = app.Flag("metrics-interval", "Collect (and post/send) metrics every specified interval.").Default("30s").Duration()
 
 	// Status & logging
-	statusAddress = app.Flag("status", "Enable serving /_status and /_metrics on given HOST:PORT (or unix:SOCKET).").PlaceHolder("ADDR").String()
-	enableProf    = app.Flag("enable-pprof", "Enable serving /debug/pprof endpoints alongside /_status (for profiling).").Bool()
-	quiet         = app.Flag("quiet", "Silence log messages (can be all, conns, conn-errs, handshake-errs; repeat flag for more than one)").Default("").Enums("", "all", "conns", "handshake-errs", "conn-errs")
+	statusAddress       = app.Flag("status", "Enable serving /_status and /_metrics on given HOST:PORT (or unix:SOCKET).").PlaceHolder("ADDR").String()
+	statusTargetAddress = app.Flag("status-target-http", "Enable specifying an HTTP target address for checking downstream healthchecks. Defaults to a TCP healthcheck if this flag is not passed.").Default("").String()
```

Comment:
I think we should call this `--target-status`, and sort it alphabetically after `--target`. That way it's clearer that it's for specifying the status endpoint for the selected target. What do you think?

Reply:
Love it! Great idea.